docs/src/main/asciidoc/authentication.adoc

```diff
@@ -42,6 +42,14 @@ A common challenge for developers is the management of secrets and credentials u
 
 We encourage using manged identity instead of using connection string or key in your application for it's more secure and will save the trouble of managing secrets and credentials. In this case, `DefaultAzureCredential` could better serve the scenario of developing locally using account information stored locally and deploying the application to Azure Cloud and using Manged Identity.
 
+==== Managed Identity Types
+There are two types of managed identities:
+
+- System-assigned Some Azure services allow you to enable a managed identity directly on a service instance. When you enable a system-assigned managed identity an identity is created in Azure AD that is tied to the lifecycle of that service instance. So when the resource is deleted, Azure automatically deletes the identity for you. By design, only that Azure resource can use this identity to request tokens from Azure AD.
+- User-assigned You may also create a managed identity as a standalone Azure resource. You can create a user-assigned managed identity and assign it to one or more instances of an Azure service. In the case of user-assigned managed identities, the identity is managed separately from the resources that use it.
```

Comment:
Suggestion:

`- System-assigned` -> `- **System-assigned**`
`- User-assigned` -> `- **User-assigned**`

![image](https://user-images.githubusercontent.com/13167207/146845373-e9d22bec-8f4c-4beb-af19-958115ef2c8b.png)

Reply:
Adding a link is better than copy text from MS docs. Because MS docs may update in the future. We should keep single source of truth.

MS docs: https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview#managed-identity-types

Reply:
I prefer to explain these concepts with minimal description, but I agree that we can link to MS Docs. But what came first to me is that jumping out of the current doc is annoying.

Reply:
Could you please add that link?